Classes/Service/RecordService.php

```diff
@@ -51,6 +51,13 @@ public function get($table, $fields, $clause = null, $groupBy = null, $orderBy =
         if ($offset) {
             $statement->setFirstResult($offset);
         }
+        
+        if (TYPO3_MODE === 'BE') {
```

Comment:
IMHO this really doesn't belong in RecordService, which is supposed to be a generic service that doesn't depend on state. If there is no alternative, we'll need to reproduce this query in Preview instead of using RecordService.

Reply:
Yes, it's not possible to edit the statement outside RecordService. Will see that soon.